src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -50,7 +50,7 @@ Typically, for most projects there are only a few transactions that you really c
 
 ### Health Checks
 
-_Health check_ type transactions, while important for checking the stability of your application, don't have any value for you beyond the task associated with them. They also tend to be high in volume, potentially using up a lot of you quota if retained. To avoid using up your volume on this type of transactions, we ignore these when sampling your data.
+_Health check_ type transactions, while important for checking the stability of your application, don't have any value for you beyond the task associated with them. They also tend to be high in volume, potentially using up a lot of your quota if retained. To avoid using up your volume on these types of transactions, we ignore these when sampling your data.
```

Comment:
```suggestion
 _Health check_ type transactions are important for checking the stability of your application, but they have no value beyond the task associated with them. They tend to be high in volume and can use up a lot of your quota if retained. To avoid this, we ignore them when sampling your data.
```